extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/sql/entity/ResultSetInformation.java

```diff
@@ -31,51 +31,27 @@
 public class ResultSetInformation
 {
 
-  @Nullable
-  private final Long numRows;
-  @Nullable
-  private final Long sizeInBytes;
-
   @Nullable
   private final ResultFormat resultFormat;
-
   @Nullable
   private final List<Object> records;
-
   @Nullable
   private final String dataSource;
+  @Nullable
+  private final List<PageInformation> pageInformationList;
 
   @JsonCreator
   public ResultSetInformation(
       @JsonProperty("resultFormat") @Nullable ResultFormat resultFormat,
-      @JsonProperty("numRows") @Nullable Long numRows,
```

Comment:
I think there is value in having the numRows and the sizeInBytes. We can rename the variables to 
numTotalRows , totalSizeInBytes